html/janus.js

```diff
@@ -1307,7 +1316,7 @@ function Janus(gatewayCallbacks) {
 					function getScreenMedia(constraints, gsmCallback) {
 						Janus.log("Adding media constraint (screen capture)");
 						Janus.debug(constraints);
-						getUserMedia(constraints,
+						navigator.mediaDevices.getUserMedia(constraints,
```

Comment:
We define getUserMedia in adapter.js to account for browser differences, what's this other method about?